sources/styles.css

```diff
@@ -484,3 +484,104 @@ section.about-container h4 {
   color: #344563;
   margin-bottom: 80px;
 }
+
+/* contact start */
+footer.contact-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 60px;
+  margin-left: 24px;
+  margin-bottom: 60px;
+  background-image: url("../images/illustration-contact-form.svg");
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position-x: 100%;
+  background-position-y: 77%;
+}
```

Comment:
- [ ] You contact section is looking good, but do not you think, that it is not aligned center? It is aligned to the right more. I tried to delete it from your code margins and it looks nice. Please, try also to play with margin and alignment to bring the design much closer to the Figma template. This is your form displaying in the browser:
<img width="301" alt="Screen Shot 2022-01-12 at 00 40 24" src="https://user-images.githubusercontent.com/49064106/149018231-27924386-200d-479d-93ce-4fabe9afdaa2.png">